### PR TITLE
remove console, features, devel from config and composer.json. Fix #15

### LINKED ISSUE #15: Fix devel settings dependency
Config sync us currently failing with the following error:
~~~
drush cim sync -y
 Collection   Config                                                         Operation                
              block.block.landinginfowithcountdownblock                      create 
              block.block.mainpagecontent                                    create 
              devel.settings                                                 create 
              mailchimp_signup.mailchimp_signup.subscribe_to_our_newsletter  create 
              system.menu.devel                                              create 
              language.negotiation                                           update 
 language.es  views.view.block_content                                       update 
 language.es  views.view.content                                             update 
 language.es  views.view.files                                               update 
 language.es  views.view.frontpage                                           update 
 language.es  views.view.user_admin_people                                   update 
 language.es  core.date_format.html_year                                     delete
Import the listed configuration changes? (y/n): y
Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization. in Drupal\Core\Config\ConfigImporter->validate() (line 728 of             [error]
/var/www/dev.drupalcamp.es/releases/20160825172101Z/web/core/lib/Drupal/Core/Config/ConfigImporter.php).
The import failed due for the following reasons:                                                                                                                               [error]
Configuration <em class="placeholder">devel.settings</em> depends on the <em class="placeholder">devel</em> extension that will not be installed after import.
~~~

Since deployment processes is cim based newer configs won't take effect until this error is removed / cc @juampynr

### PR DESCRIPTION
**Pull request by nuez**
Removed unused dependencies from composer and config.

-features
-devel

I also removed console, as it seems to cause a conflict (doesn't work for me, where the globally installed drupal console does).

### CODE REVIEW

File: config/language/es/views.view.frontpage.yml [hunk 1]
```diff
@@ -21,7 +22,7 @@ display:
             items_per_page_options_all_label: '— Todo(s) —'
             offset_label: Desplazamiento
           tags:
-            previous: '‹ Previous'
+            next: 'Siguiente ›'
```
Comment (juampynr) [inline config/language/es/views.view.frontpage.yml]: Why Spanish, @nuez? Let's keep everything in English and then we can translate. Deal?

**Comment by NITEMAN** [comment]
If this fix #15 , please close the issue when merged

File: config/language/es/views.view.frontpage.yml (hunk 1 above)
Comment (nuez) [inline config/language/es/views.view.frontpage.yml]: Of course @juampynr we have a deal...As you can imagine this wasn't intentional, Drupals apparently CMI still has some issues. It seems exports strings from Spanish when spanish is negotiated. I'll look into it. In any case, I'm not a big fan of having that one config.tar.gz and will open another issue to discuss the Configuration Managment, which to me seems a bit of a hassle the way it's working right now.

**Comment by nuez** [comment]
Still having to figure out the best way to work with the config. I've updated the config files from drush this time, and manually tarred them into config.tar.gz

**Comment by juampynr** [comment]
Woops, @nuez, can you look at the above conflicts?

**Comment by nuez** [comment]
OK done

File: config/language/es/views.view.frontpage.yml (hunk 1 above)
Comment (juampynr) [inline config/language/es/views.view.frontpage.yml]: Sure thing!

**Pull request merged**